scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Database.scala

```diff
@@ -34,6 +34,12 @@ import scala.meta.{semantic => m}
 
 object Database {
   def load(classpath: Classpath, sourcepath: Sourcepath): Database = {
-    v.Database.load(classpath).toSchema.toMeta(sourcepath)
+    v.Database.load(classpath).toSchema.toMeta(Some(sourcepath))
+  }
```

Comment:
How about we add another overload that takes `(classpath: Classpath)`, i.e operates exactly like `Array[Byte]`, but without its users having to know things like `META-INF`?